doc/.gitignore

```diff
@@ -1,11 +1,9 @@
-build
 /html
 
-/builtins
-/modules
-/primers
-/examples
+/rst/builtins
+/rst/modules
+/rst/primers
+/rst/examples
 
-tools/chpldoc/man.rst
-tools/chplvis/examples
-usingchapel/man.rst
+man.rst
```

Comment:
This entry looks wrong.  At least, I don't have a doc/man.rst (and wouldn't expect to).